bazel/repositories.bzl

```diff
@@ -20,11 +20,9 @@ def cartographer_repositories():
     _maybe(
         http_archive,
         name = "com_github_nelhage_rules_boost",
-        sha256 = "371f49e7b29e44a718baf8b9a2dd3eca865005a851c9ecf8fb6a10a715aa58dd",
-        strip_prefix = "rules_boost-a5a95642f6097f8949020646ffe89d7243008981",
+        strip_prefix = "rules_boost-c3fae06e819ed8b93e31b150387dce4864758643",
```

Comment:
Can you add `sha256 = "f7d620c0061631d5b7685cd1065f2e2bf0768559555010a75e8e4720006f5867"`? Omitting it is a security risk, see https://bazel.build/rules/lib/repo/http.

Reply:
done.